crates/ruma-events/src/room/message/audio.rs

```diff
@@ -124,6 +124,14 @@ pub struct UnstableAudioDetailsContentBlock {
     pub waveform: Vec<Amplitude>,
 }
 
+#[cfg(feature = "unstable-msc3245-v1-compat")]
+impl UnstableAudioDetailsContentBlock {
+    /// Creates a new `UnstableAudioDetailsContentBlock ` with the given duration and waveform
+    pub fn new(duration: Duration, waveform: Vec<Amplitude>) -> Self {
```

Comment:
In the `Cargo.toml` of `ruma-events` I found this line:
```
unstable-msc3245-v1-compat = ["unstable-msc3246"]
```

Does it mean we already have this dependency so that every time `unstable-msc3245-v1-compat` also `unstable-msc3246` is on?
If so, do we still need a change here?

Reply:
Oh you're right, then could you remove the feature flag on the field at line 122? It shouldn't be there in the first place.

Reply:
It make sense. Applied -> https://github.com/ruma/ruma/pull/1680/commits/dcd2034720a9a5d178ed945c6a1bc371a3509d4e